packages/config/config/devices/0x0371/zwa012.json

```
@@ -47,52 +63,142 @@
 			"$import": "~/0x0086/templates/aeotec_template.json#sensor_operation_mode"
 		},
 		"2": {
-			"$import": "~/0x0086/templates/aeotec_template.json#invert_state_report"
+			"$import": "~/0x0086/templates/aeotec_template.json#external_switch_3way",
+			"label": "Dry Contact Operation Mode",
+			"maxValue": 3,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Switch mode",
+					"value": 0
+				},
+				{
+					"label": "Switch mode inverted",
+					"value": 1
+				},
+				{
+					"label": "Momentary mode",
+					"value": 2
+				},
+				{
+					"label": "Momentary mode inverted",
+					"value": 3
+				}
+			]
 		},
-		"3[0x02]": {
+		"3": {
+			"$import": "~/0x0086/templates/aeotec_template.json#invert_state_report",
```

```suggestion
			"$import": "~/templates/master_template.json#base_0-1_nounit",
```
Don't import a second-level template if you're overwriting everything it changes from the original template.